server/service/hosts.go

```diff
@@ -1410,7 +1410,6 @@ func hostsReportEndpoint(ctx context.Context, request interface{}, svc fleet.Ser
 		return hostsReportResponse{Err: err}, nil
 	}
 
-	req.Opts.DisableFailingPolicies = false
```

Comment:
curious what that was doing there in the first place 🤔

Reply:
It was not allowing the user to execute `/hosts/report` with `disable_failing_policies`. Now by default it will behave the same (`disable_failing_policies=false`), but the user can decide to not calculate the failing policies with `disable_failing_policies=true`

E.g. the `/hosts/report?columns=id,hostname` endpoint allows getting a subset of the host fields (id and hostname in this case), and maybe the user is not interested on failure count policies, so they can disable that (and improve performance) with `disable_failing_policies=true`.